components/list/list-item-button-mixin.js

```diff
@@ -4,13 +4,30 @@ import { getUniqueId } from '../../helpers/uniqueId.js';
 import { ListItemMixin } from './list-item-mixin.js';
 
 export const ListItemButtonMixin = superclass => class extends ListItemMixin(superclass) {
+	static get properties() {
+		return {
+			/**
+			 * Disables the primary action button
+			 * @type {boolean}
+			 */
+			buttonDisabled : { type: Boolean, attribute: 'button-disabled' }
+		};
+	}
 
 	static get styles() {
 
 		const styles = [ css`
-			:host {
+			:host(:not([button-disabled])) {
 				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
 			}
+			:host([button-disabled][_hovering-primary-action]) .d2l-list-item-content,
+			:host([button-disabled][_focusing-primary-action]) .d2l-list-item-content {
+				--d2l-list-item-content-text-color: var(--d2l-color-ferrite);
```

Comment:
```suggestion
				--d2l-list-item-content-text-color: unset;
```

Current list-items don't have a default color, they inherit it from their parent